tests/unit/hive_metastore/test_migrate_acls.py

```diff
@@ -0,0 +1,198 @@
+import logging
+from unittest.mock import create_autospec
+import pytest
+from databricks.labs.lsql.backends import MockBackend
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.ucx.account.workspaces import WorkspaceInfo
+from databricks.labs.ucx.hive_metastore.grants import Grant, GrantsCrawler, PrincipalACL
+from databricks.labs.ucx.hive_metastore.table_migrate import (
+    ACLMigrator,
+)
+from databricks.labs.ucx.hive_metastore.migration_status import (
+    MigrationStatusRefresher,
+    MigrationIndex,
+)
+from databricks.labs.ucx.hive_metastore.tables import (
+    TablesCrawler,
+)
+from databricks.labs.ucx.hive_metastore.udfs import UdfsCrawler
+from databricks.labs.ucx.workspace_access.groups import GroupManager
+
+from .. import GROUPS
+from ..workspace_access.test_tacl import UCX_TABLES
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def ws():
+    client = create_autospec(WorkspaceClient)
+    client.get_workspace_id.return_value = "12345"
+    return client
+
+
+@pytest.fixture
+def ws_info():
+    client = create_autospec(WorkspaceInfo)
+    client.current.return_value = "hms_fed"
+    return client
```

Comment:
```suggestion
```
these are used only once and don't improve readability. inline them.

Reply:
They are used 3 times each. Still remove?